parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -253,17 +253,19 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
 fi
 {{end}}
 
+VALIDATION_ERR=0
+
 {{- if IsHostedMaster }}
 RES=$(retrycmd_if_failure 20 1 3 nslookup ${API_SERVER_NAME})
 STS=$?
 if [[ $STS != 0 ]]; then
-  if [[ $RES == *"168.63.129.16"* ]]; then
-    exit {{GetCSEErrorCode "ERR_K8S_API_SERVER_AZURE_DNS_LOOKUP_FAIL"}}
-  else
-    exit {{GetCSEErrorCode "ERR_K8S_API_SERVER_CONN_FAIL"}}
```

Comment:
actually found this error code was somehow wrong

Reply:
yeah it's not actualyl the api service connection it's your non azure dns that can't find the api server ip.